packages/luna/src/components/Chat/ChatRoom/ChatRoom.tsx

```diff
@@ -68,70 +86,96 @@ export const ChatRoom: React.FC<{
         }
       )}
     >
-      <div className="tw-w-14 tw-h-14 tw-rounded-full tw-overflow-hidden">
-        <Avatar src={image} alt={name} seed={userId.toString()} />
+      <div className="tw-w-14 tw-h-14 tw-relative tw-rounded-full tw-shrink-0">
+        <div className="tw-overflow-hidden tw-rounded-full tw-relative tw-w-full tw-h-full">
+          <Avatar src={image} alt={name} seed={userId.toString()} />
+        </div>
+        {online && !isActive ? (
+          <div className="tw-absolute tw-bottom-0 tw-right-1 tw-z-activityIndicator tw-w-3 tw-h-3 tw-rounded-full tw-bg-brand-500 tw-border-[.5px] tw-border-natural-50 " />
```

Comment:
Use the dash-case naming notation in the class names. 

```suggestion
          <div className="tw-absolute tw-bottom-0 tw-right-1 tw-z-online-indicator tw-w-3 tw-h-3 tw-rounded-full tw-bg-brand-500 tw-border-[.5px] tw-border-natural-50 " />
```